sdk/identity/azure-identity/azure/identity/_credentials/environment.py

```diff
@@ -75,6 +78,7 @@ def get_token(self, *scopes, **kwargs):  # pylint:disable=unused-argument
         # type: (*str, **Any) -> AccessToken
         """
         Request an access token for `scopes`.
+
```

Comment:
Side note, in theory the summary should be right after the """, without a new line:
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

```
Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line. The summary line may be on the same line as the opening quotes or on the next line. 
```

Reply:
Thanks for pointing this out, I'll reformat everything before the next release (#7663)